### PR TITLE
Fix links from FAQ to blog (Addresses #2609)

### LINKED ISSUE #2609: Most links to the blog from the FAQ are broken
## Where to find the issue
- https://www.coronawarn.app/de/faq/
- https://www.coronawarn.app/en/faq/

## Describe the issue
All links from the FAQ to the blog are broken. Try e.g. https://www.coronawarn.app/en/faq/results/#check_in_luca_scan. You will only get a 404. 

## Suggested change
Fix the issue.

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR fixes an issue which caused the links from the FAQ to the blog to fail and result in a 404 page.

## What was changed

Many FAQ entries which included invalid links to the blog.

## Screenshots

No visible changes here. 

---

**Closes #2609**

### CODE REVIEW

**Comment by MikeMcC399** [comment]
-This does not fix all blog links issues. 
- I suggest to fix issues from #2611 in one PR covering all relative link issues.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Which blog links did I miss?

I can certainly extend this PR to other links too, but I rather want this to get merged ASAP and then tackle the other issues in another PR.

File: src/data/faq.json [hunk 1]
```diff
@@ -2308,7 +2308,7 @@
                                 "anchor": "numbers_RKI",
                                 "active": false,
                                 "textblock": [
-                                    "An overview of key figures for the Corona-Warn-App is available in the form of a <a href='/en/analysis/' target='_blank'>digital dashboard</a>. The app displays selected statistics including <a href='/en/blog/2021-08-05-statistictiles/#tile-warnings-by-app-users' target='_blank'>Warnings by App Users</a>.",
+                                    "An overview of key figures for the Corona-Warn-App is available in the form of a <a href='/en/analysis/' target='_blank'>digital dashboard</a>. The app displays selected statistics including <a href=2021-08-05-statistictiles/#tile-warnings-by-app-users' target='_blank'>Warnings by App Users</a>.",
```
Comment (Ein-Tim) [inline src/data/faq.json:2311]:
```suggestion
                                    "An overview of key figures for the Corona-Warn-App is available in the form of a <a href='/en/analysis/' target='_blank'>digital dashboard</a>. The app displays selected statistics including <a href=../../blog/2021-08-05-statistictiles/#tile-warnings-by-app-users' target='_blank'>Warnings by App Users</a>.",
```

File: src/data/faq.json [hunk 2]
```diff
@@ -3061,7 +3061,7 @@
                 {
                     "term": "Data Donation",
                     "anchor": "data_donation",
-                    "description": "A voluntary and anonymous way to provide the RKI with data on the use of the CWA. Voluntary data donation can help experts evaluate the effectiveness of the app and further improve it. More information is summarized here: <a href='/en/blog/2021-03-04-corona-warn-app-version-1-13/'>Science Blog: Users can provide data voluntarily to help improve the Corona-Warn-App further</a>."
+                    "description": "A voluntary and anonymous way to provide the RKI with data on the use of the CWA. Voluntary data donation can help experts evaluate the effectiveness of the app and further improve it. More information is summarized here: <a href=2021-03-04-corona-warn-app-version-1-13/'>Science Blog: Users can provide data voluntarily to help improve the Corona-Warn-App further</a>."
```
Comment (Ein-Tim) [inline src/data/faq.json:3064]:
```suggestion
                    "description": "A voluntary and anonymous way to provide the RKI with data on the use of the CWA. Voluntary data donation can help experts evaluate the effectiveness of the app and further improve it. More information is summarized here: <a href=../../blog/2021-03-04-corona-warn-app-version-1-13/'>Science Blog: Users can provide data voluntarily to help improve the Corona-Warn-App further</a>."
```

File: src/data/faq_de.json [hunk 3]
```diff
@@ -1294,7 +1294,7 @@
...
                                     "Hinweise:<ul><li>Die Fehlerberichte werden nach 14 Tagen automatisch gelöscht</li><li>Berichte, die unaufgefordert an den Server übertragen werden, können durch die Entwickler nicht analysiert werden</li><li>Für die Fehlerberichte gibt es keinen direkten Rückkanal in der Corona-Warn-App. Der technische Support kann nur über einen der oben genannten Wege mit Ihnen kommunizieren</li></ul>",
                                     "Weitere Infos sowie Screenshots finden Sie im Blog-Artikel <a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank'>Corona-Warn-App Version 2.2 ermöglicht die Erstellung von Schnelltest-Profil und Fehlerbericht</a>."
```
Comment (Ein-Tim) [inline src/data/faq_de.json:1299]:
```suggestion
                    "Weitere Infos sowie Screenshots finden Sie im Blog-Artikel <a href='../../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank'>Corona-Warn-App Version 2.2 ermöglicht die Erstellung von Schnelltest-Profil und Fehlerbericht</a>."
```

File: src/data/faq_de.json [hunk 4]
```diff
@@ -2261,9 +2261,9 @@
...
+                                    "<ul><li><a href='../blog/2021-02-23-corona-warn-app-risk-calculation-optimization/' target='_blank' >23.02.2021 - Risikoberechnung der Corona-Warn-App nach detaillierten Tests weiter angepasst</a></li><li><a href='../../blog/2021-03-19-risk-calculation-improvement/' target='_blank' >19.03.2021 - Die Risikoberechnung der Corona-Warn-App wird weiter verbessert</a></li><li><a href='../../blog/2021-04-16-corona-warn-app-risk-calculation-further-improved/' target='_blank' >16.04.2021 - Das Projektteam verbessert die Risikoberechnung der Corona-Warn-App in Reaktion auf die Infektionslage</a></li></ul></ul>",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:2266]:
```suggestion
                                    "<ul><li><a href='../../blog/2021-02-23-corona-warn-app-risk-calculation-optimization/' target='_blank' >23.02.2021 - Risikoberechnung der Corona-Warn-App nach detaillierten Tests weiter angepasst</a></li><li><a href='../../blog/2021-03-19-risk-calculation-improvement/' target='_blank' >19.03.2021 - Die Risikoberechnung der Corona-Warn-App wird weiter verbessert</a></li><li><a href='../../blog/2021-04-16-corona-warn-app-risk-calculation-further-improved/' target='_blank' >16.04.2021 - Das Projektteam verbessert die Risikoberechnung der Corona-Warn-App in Reaktion auf die Infektionslage</a></li></ul></ul>",
```

**Review by Ein-Tim — COMMENTED** [review]
Some issues spotted

**Comment by MikeMcC399** [comment]
@Ein-Tim 

- If you use an updated `check_links.js` (see https://github.com/corona-warn-app/cwa-website/pull/2614) you can check for remaining problems. Hopefully the Open Source Team will pick up this PR quickly and merge it.
- I will run the check on your branch and post the results here.
- You might like to put this PR into Draft status whilst you are still working on making corrections to it

**Comment by MikeMcC399** [comment]
```
==================== Broken links ====================

[RESPONSE 404] http://localhost:8000/de/faq/eventregistration on '/de/faq/results/' 
[RESPONSE 404] http://localhost:8000/de/faq/blog/2021-05-12-corona-warn-app-version-2-2/ on '/de/faq/results/' 
[RESPONSE 404] http://localhost:8000/de/faq/blog/2020-12-15-ios-12-5 on '/de/faq/results/' 
[RESPONSE 404] http://localhost:8000/de/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage6.pdf on '/de/faq/results/' 
[RESPONSE 404] http://localhost:8000/de/faq/blog/2021-02-23-corona-warn-app-risk-calculation-optimization/ on '/de/faq/results/' 
[RESPONSE 404] https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests-01-12-2020.pdf?__blob=publicationFile&v=7 on '/de/faq/results/' 
[RESPONSE 404] http://localhost:8000/en/faq/eventregistration on '/en/faq/results/' 
[RESPONSE 404] http://localhost:8000/en/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage6.pdf on '/en/faq/results/' 
[RESPONSE 404] http://localhost:8000/en/faq/results/2021-03-04-corona-warn-app-version-1-13/' on '/en/faq/results/' 
[RESPONSE 404] https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests-01-12-2020.pdf?__blob=publicationFile&v=7 on '/en/faq/results/' 
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
@Ein-Tim 
Please check with updated `check_links.js` before proceeding.
You already identified some changes which you intend to make yourself in any case.

Let me know if you want help in cross-checking.

**Comment by Ein-Tim** [comment]
Setting to draft until remaining problems are solved.

File: src/data/faq_de.json (hunk 4 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Resolved.

File: src/data/faq_de.json (hunk 3 above)
Comment (Ein-Tim) [reply src/data/faq_de.json:1299]: Resolved.

File: src/data/faq.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Resolved.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Resolved.

**Comment by Ein-Tim** [comment]
@MikeMcC399 I fixed some more issues, however, I did not find the issue with the link to the iOS 12.5 blog post. I will check again.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Could you maybe run the script again? I‘m currenlty not on my main device and thus have to use GitHubs web interface.

**Comment by Ein-Tim** [comment]
Closing in favour of https://github.com/corona-warn-app/cwa-website/pull/2617.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

- I was just about to run the script when I saw that you had closed this. 
- #2617 is more complete and, like you, I think that the problems should be solved as quickly and as comprehensively as possible
- Thanks for pointing out there were missing changes in #2617. You were right that there were some language dependent links which I missed on the first edit. I made these language independent.

I hope that there aren't any more hidden link problems. We need the `check_anchor_links.js` to be modified in order to be sure.

If you see anything else wrong in #2617 please add a comment there.

**Pull request closed without merging**